department-of-veterans-affairs/vets-api
# ARF-83474 pt. 1 Create table for new PilotRepresentative model

## Summary

- This work is behind a feature toggle (flipper): NO
- This PR adds a new table for the new `AccreditedRepresentativesPortal::PilotRepresentative` model.
- This new model will replace the old `AccreditedRepresentativesPortal::VerifiedRepresentative` model with a new model with a more appropriate name.
- Benefits Accredited Rep Facing team. Yes, we own maintenance of this.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/83474

## Testing done
- No unit tests needed for a DB migration.
- Test steps:
  - Check out this branch and start vets-api and its dependencies.
  - Run `rails db:migrate` to verify that the migration runs without errors.
  - Run `rails db:rollback` to verify the migration can be reversed.


## Screenshots
N/A

## What areas of the site does it impact?
This table is not being used at the moment, so no impact at this point.

## Acceptance criteria

- [N/A]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [N/A]  Events are being sent to the appropriate logging solution
- [N/A]  Documentation has been updated (link to documentation)
- [N/A]  Feature/bug has a monitor built into Datadog (if applicable)
- [N/A]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [N/A]  I added a screenshot of the developed feature

## Requested Feedback

Review that all DB changes are appropriate.
